requirements/devel.txt

```diff
@@ -11,4 +11,8 @@ coverage==6.2
 tox-current-env==0.0.11
 # pytest dependency
 pytest-django==4.5.2
-pytest-redis==2.4.0
\ No newline at end of file
+pytest-redis==2.4.0
+# integration
+curlify
```

Comment:
Please provide the detail version information.

Reply:
Updated.